src/Mod/Spreadsheet/Gui/SheetTableView.cpp

```diff
@@ -182,6 +240,19 @@ void SheetTableView::insertRows()
     Gui::Command::doCommand(Gui::Command::Doc, "App.ActiveDocument.recompute()");
 }
 
+void SheetTableView::insertRowsAfter()
+{
+    assert(sheet != 0);
+    const auto rows = selectionModel()->selectedRows();
+    const auto & [min, max] = selectedMinMaxRow(rows);
+    assert(max - min == row.size() - 1);
```

Comment:
I think you mean `rows` here -- this line doesn't compile right now.

Reply:
Well this is embarrassing! After resurrecting this old branch of mine, I only compiled it in release mode, which strips away all assertions. Thanks for pointing it out